docs/Contributing/Release.md

```diff
@@ -0,0 +1,143 @@
+# Release Process
+
+
+## Prerequisites:
+- Prepare release notes
+
+
+## Weekly chores
+### Enarx dependency update
+- Update local code
+  ```bash
+  git fetch --all
+  git checkout main
+  git pull --ff-only
+  ```
+- Checkout chore branch
+  ```bash
+  git checkout -b update-prerequisites
+  ```
+- Check to see `Cargo.toml` is refering to the latest updated prerequisite crates
+- Run cargo-update on all individual sub-crates in updated packages are referenced
+   ```bash
+   for d in internal/*/ ; do (cd "$d" && cargo update); done
+   ```
+- Git commit sub-crate update
+  ```bash
+   git commit -asm 'Updated Enarx sub-crate dependencies'
+   ```
+- Run cargo update on `enarx`
+  ```bash
+  cargo update
+  ```
+- Git commit sub-crate update
+  ```bash
+   git commit -asm 'Updated Enarx dependencies'
+   ```
+- Run build and tests
+  ```bash
+  cargo clean
+  cargo build
+  cargo tests
+  ```
+- Create PR
+  ```bash
+  git push origin update-prerequisites
+  ```
+
+
+## Enarx Release
+
+### Update and release prerequiste crates (this may be an optional step)
+This includes `xsave`, `sallyport`, etc crates:
+- Ensure all approved PRs are merged
+- Get latest updates and checkout branch
+  ```bash
+  git fetch --all
+  git checkout main
+  git pull --ff-only
+  git checkout -b update/deps
+  ```
+- Update dependencies
+  ```bash
+  cargo update
+  ```
+- Determine if crate builds and if it works
+  ```bash
+  cargo clean
+  cargo test
+  ```
+- Determine version bump by reviewing output of `git log`
+- Update and bump version in `Cargo.toml`
+- Run `cargo test` again
+- Commit change and push to repo
+  ```bash
+  git commit -asS -m 'Release vX.X.X'
+  git push origin
+  ```
+- Confirm that changes passed on CI
+- Create a git tag
+  ```bash
+  git tag vX.X.X
+  git push --tags origin vX.X.X
+  ```
+- Cargo publish
+  ```bash
+  cargo publish
+  ```
+
+### The Enarx release itself
+- Get latest updates and checkout branch
+  ```bash
+  git fetch --all
+  git checkout main
+  git pull --ff-only
+  git checkout -b release-<my fancy new version>
+  ```
+- Bump version inside sub-crate `internal/{shim-sev,shim-sgx,wasmdr}/Cargo.toml` files
+  ```bash
+  export OLD_VERSION=$(grep '^version =' Cargo.toml | cut -d'=' -f 2)
+  export NEW_VERSION='"<my fancy new version>"'
+  for d in internal/*/ ; do (cd "$d" \
+    sed -i \""s/^version = ${OLD_VERSION}/version = ${NEW_VERSION}/"\" Cargo.toml \
+    cargo update -p $(basename ${d}) \
+    git mv $i ${i%.toml}.tml ) \
+  done
+  ```
+- Run unittests
+  ```bash
+  cargo clean
+  cargo test
+  ```
+- Check cargo manifest
+  ```bash
+  cargo package --allow-dirty -l
+  ```
+- Commit change and push to repo
+  ```bash
+  git commit -asS -m "Release v${NEW_VERSION}"
```

Comment:
see above

Reply:
Updated commit string to comply with conventional commits.